common/xrdp_sockets.h

```diff
@@ -26,8 +26,8 @@
 #define CHANSRV_PORT_OUT_BASE_STR  "xrdp_chansrv_audio_out_socket_%d"
 #define CHANSRV_PORT_IN_BASE_STR   "xrdp_chansrv_audio_in_socket_%d"
 #define CHANSRV_API_BASE_STR       "xrdpapi_%d"
-#define XRDP_X11RDP_BASE_STR       "xrdp_display_%d"
-#define XRDP_DISCONNECT_BASE_STR   "xrdp_disconnect_display_%d"
+#define XRDP_X11RDP_BASE_STR       "xrdp_disp_%d"
+#define XRDP_DISCONNECT_BASE_STR   "xrdp_disconnect_disp_%d"
 #define SCP_LISTEN_PORT_BASE_STR   "sesman.socket"
```

Comment:
Is there a good reason to change these?

If so fine, but bear in mind you'll also need to update https://github.com/neutrinolabs/xrdp/wiki/The-socketdir-directory where these are documented.

Reply:
I don't know how I got this, but I specifically remember messing with `make distcheck` and getting an error that there was a risk that this specific string could exceed the length of sa.sun_path on [this line](https://github.com/neutrinolabs/xrdp/blob/dd03a1ad10f4d4dd65c82d545aefa3d9d68005a6/sesman/tools/dis.c#L65), but I can no longer reproduce it, nor do I understand why I got this error in the first place. Ignore this, I won't check it in.

Reply:
Interesting - thanks for the reference.

According to [unix(7)](https://www.man7.org/linux/man-pages/man7/unix.7.html), `sa.sun_path` will be at least 92 bytes. POSIX isn't as definitive and says it is [typically between 92 and 108 bytes](https://pubs.opengroup.org/onlinepubs/009696699/basedefs/sys/un.h.html). Either should be fine for `XRDP_DISCONNECT_STR`

We should really be using `g_snprintf()` to bound this write, but given where it is I don't think it's a big concern.

Reply:
Agreed. Resolving.